lua/neo-tree/ui/inputs.lua

```diff
@@ -91,12 +91,7 @@ M.confirm = function(message, callback)
     input.prompt_type = "confirm"
     M.show_input(input)
   else
-    local opts = {
-      prompt = message .. " y/n: ",
-    }
-    vim.ui.input(opts, function(value)
-      callback(value == "y" or value == "Y")
-    end)
+    callback(vim.fn.confirm(message, "&y[Y]\n&n[N]"))
```

Comment:
This does not work at all. It will confirm like "yes" for any choice. You need to test your code before you submit a PR.

Also, I think `"&Yes\n&No"` makes for a better set of choices.

I think this will work:
```suggestion
    callback(vim.fn.confirm(message, "&Yes\n&No") == 1)
```

Reply:
Sorry for the imprudence，and thanks for your patience.
It does need the `==` to handle for different inputs, as selecting `no` returns `2`, and <C-c> returns `0`.